db/point_accrual.py

```diff
@@ -21,6 +21,8 @@
     int(Config.CONFIG["Discord"]["GiftedTier3RoleID"]): 4,
 }
 
+SIX_AM_PST = 14  # 6am PST in UTC
```

Comment:
we should use `zoneinfo` to do actual time zones since daylight savings is a thing

Reply:
Good call - I don't think it will matter in practice, but it certainly is cleaner. The idea is that these messages have to take place in #stream-chat, which is manually locked and unlocked during stream time. So technically a message could only come in during that additional hour if it was during stream, but his is pretty ugly as is